### ISSUE TITLE
Onnx support

### ISSUE THREAD

**Issue by Flova** [bhuman/CompiledNN#20]
Currently, the loading ONNX files results in the following output:
```
/tmp/CompiledNN/Src/CompiledNN/Formats/ONNX.cpp:37: FAIL: Not yet implemented.
```

ONNX would be cool as it is not so dependent on the training framework and some code seems to be there.

**Comment by ahasselbring** [comment]
Yes, that has been an "internal" ticket for a long time, but I never really get around to it.

Generally, we hope that we can get rid of CompiledNN soon anyway because there should be some other industry-backed tool that can replace it.

**Comment by Flova** [comment]
Okay, it is also not that important for me. I just wanted to run our network with the compiler to compare the performance to other approaches.
